### ISSUE TITLE
helpers.money -> helpers.currency?

### ISSUE THREAD

**Issue by aslo** [isocket/cortex-helpers#2]
We now have the isocket-cortex-currency module to handle currency display.  Currently it only has one function, `Currency.format()` which is like a more generalized `helpers.money`. 

Can we save a lot of code by just extending helpers.money to provide support for the few currencies that we need? That plus adding a `helpers.unmoney` function *could* remove the need to cortex-currency altogether!

**Comment by maxbeatty** [comment]
I didn't think anyone was still using `unmoney` so I removed it in the v1. Lemme know if I missed something. 

Would [accounting.js](http://openexchangerates.github.io/accounting.js/) do what we need to handle money and currency? iris and sclera already use it in addition to `helpers.money`. I'd prefer to offload responsibility to accounting.js, moment.js, and lodash any chance we get so we have less to maintain.

**Comment by aslo** [comment]
Yep, accounting.js gets us most of the way there; actually, cortex-currency depends internally on accounting.js. All it does is combine acccounting.js + currency data to provide a simple interface for displaying money that comes from the service layer (i.e. an amount + a currency code).

When I first built it, the requirements were murky - so it's super general. But now it's clear that we only need to support like 4 currency types. We also now know what the "official" rubicon money format is. So, we can probably do away with cortex-currency, but I still think it would be really nice to have an internal helper that gets us from 
```
money = 
  amount: 150.22
  currency: 'AUD'
```
to 
```
"A$150.22" # i think this is how it's supposed to be displayed
```

**Comment by chan-dra** [comment]
Weighing in here:
I think that adding cortex-currency is much better than having each app handle their currency
- Will be a centralized place for handling currency
- Very less refactor (or none) from apps to move from current helpers.money
- Consistent across all apps.

On a side note:
I suggest we refactor [this](https://github.com/isocket/isocket-cortex-currency/blob/master/src/data/currencies.json#L896-L901) to something like:
```
USD: {
   code: '$', # USD -> $
   decimals: 2
   .... 
}
```
so `currency.format(1234)` returns back with `$1,234.00` instead of `USD 1,234.00`
